modules/atf/util.lua

```diff
@@ -63,6 +63,19 @@ function table.removeKey(t, k)
   return a
 end
 
+function check_required_fields()
+  if (not is_file_exists(config.pathToSDL.."smartDeviceLinkCore")) and 
+     (not is_file_exists(config.pathToSDL.."/smartDeviceLinkCore")) then
+    print("ERROR: SDL is not accessible at the specified path: "..config.pathToSDL)
+    os.exit(1)
+  end
+  if (not is_file_exists(config.pathToSDLInterfaces.."MOBILE_API.xml")) and 
+     (not is_file_exists(config.pathToSDLInterfaces.."/MOBILE_API.xml")) then
```

Comment:
@vlantonov , This check is for correct path to *.xml interfaces, to omit errors with wrong or empty path. Actually I can add check for HMI_API, but I don't think that is necessary